### ISSUE TITLE
make build Error

### ISSUE THREAD

**Issue by akdk2001** [iron-io/functions#688]
I am trying to build iron functions code. 
But there are some issues.

gwontaegmin-ui-MacBook-Pro:functions taekmin$ make build
go build -o functions
api/runner/task.go:15:2: cannot find package "github.com/moby/moby/cli/config/configfile" in any of:
	/usr/local/go/src/github.com/moby/moby/cli/config/configfile (from $GOROOT)
	/Users/taekmin/go/src/github.com/moby/moby/cli/config/configfile (from $GOPATH)
../../Sirupsen/logrus/hooks/syslog/syslog.go:10:2: case-insensitive import collision: "github.com/sirupsen/logrus" and "github.com/Sirupsen/logrus"
api/server/server.go:25:2: code in directory /Users/taekmin/go/src/github.com/ucirello/supervisor expects import "cirello.io/supervisor"
make: *** [build] Error 1

So, I tried to install that package. but that package can't be installed.

gwontaegmin-ui-MacBook-Pro:functions taekmin$ go get -v github.com/moby/moby/cli/config/configfile
github.com/moby/moby (download)
package github.com/moby/moby/cli/config/configfile: cannot find package "github.com/moby/moby/cli/config/configfile" in any of:
	/usr/local/go/src/github.com/moby/moby/cli/config/configfile (from $GOROOT)
	/Users/taekmin/go/src/github.com/moby/moby/cli/config/configfile (from $GOPATH)
gwontaegmin-ui-MacBook-Pro:functions taekmin$ 

let me know the solution.

**Comment by akdk2001** [comment]
I was searching, I saw the same issue posted in last Nov. 
Contributor seemed to try to fix the issue.
How is it going on? I wonder when branch, fix-deps is going to merge with master state and that issues are closed?

**Comment by c0ze** [comment]
@TaekminKwon unfortunately I couldn't fix the issue, and these days I cant have the time to attempt a fix.

Would like to give it a try ?

**Comment by akdk2001** [comment]
@c0ze I downloaded ironfunctions by brew install command. And I confirmed it works well. (tried Quick start)
How does it work with the issue?

**Comment by c0ze** [comment]
yes. up unto a certain point, we were able to build the package by using the dep manifest in the repo by doing `make dep`.

Right now, I am not sure if it works any more. So we need to rebuild dependencies from scratch I think.
